engine/lite/request-response/runtime/src/main/scala/pl/touk/nussknacker/engine/requestresponse/RequestResponseConfig.scala

```diff
@@ -13,4 +13,7 @@ object OpenApiDefinitionConfig {
 
 }
 
-case class RequestResponseConfig(definitionMetadata: OpenApiDefinitionConfig = OpenApiDefinitionConfig())
+case class BasicAuthConfig(user: String = "publisher", password: String)
```

Comment:
Don't think publisher is a good name for req-res :) Let's leave it blank

Reply:
changed